### ISSUE TITLE
[UWP][Accessibility] Visual difference between selected and non-selected value in compact choice set

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#4422]
# Platform

* UWP

# Details

From Inclusive and Accessible Design Spec Review Report:

Using file Input.ChoiceSet.Compact.Label.json: There should be clear visual difference of selected value and not selected value in a drop down for both required and non-required fields.
